### PR TITLE
Remove unnecessary `@Suppress("ReturnCount")`

### PR DESCRIPTION
**Pull request by BraisGabin**
This is doable thanks to #5835

### CODE REVIEW

File: detekt-rules-style/src/main/kotlin/io/gitlab/arturbosch/detekt/rules/style/RedundantExplicitType.kt [hunk 1]
```diff
@@ -58,8 +58,7 @@ class RedundantExplicitType(config: Config) : Rule(config) {
     override fun visitProperty(property: KtProperty) {
         if (!property.isLocal) return
         val typeReference = property.typeReference ?: return
-        val type =
-            (bindingContext[BindingContext.VARIABLE, property])?.type ?: return
+        val type = (bindingContext[BindingContext.VARIABLE, property])?.type ?: return
```
Comment (3flex) [inline detekt-rules-style/src/main/kotlin/io/gitlab/arturbosch/detekt/rules/style/RedundantExplicitType.kt:61]: Should the ReturnCount suppression be removed from this function now? This change looks unrelated if not - also I'm curious if this had to be reformatted so the suppression can be removed?

**Review by 3flex — APPROVED** [review]

**Comment by 3flex** [comment]
Nice!

File: detekt-rules-style/src/main/kotlin/io/gitlab/arturbosch/detekt/rules/style/RedundantExplicitType.kt (hunk 1 above)
Comment (BraisGabin) [reply detekt-rules-style/src/main/kotlin/io/gitlab/arturbosch/detekt/rules/style/RedundantExplicitType.kt:61]: This is just a Boy scout change. It has not relation with what this rule archives.
